src/rtapi/rtapi_compat.c

```diff
@@ -246,11 +243,6 @@ flavor_ptr flavor_byid(int flavor_id)
 flavor_ptr default_flavor(void)
 {
     char *fname = getenv("FLAVOR");
```

Comment:
not to be done in this part, but we should check the return value against NULL and use the compile time value as default if not present